editoast/Cargo.toml

```diff
@@ -26,21 +26,21 @@ diesel = { version = "2.0", features = [
 diesel_json = "0.2.0"
 image = "0.24"
 json-patch = "1.0"
-pathfinding = "4.2.1"
+pathfinding = "4.3"
```

Comment:
just out of curiosity, is there some rule when to bind to the patch version, and when to the minor version?

Reply:
Mainly if I trust the crate not to release a patch version that breaks the API.